webapp/channels/src/components/widgets/menu/menu_items/submenu_item.test.tsx

```diff
@@ -81,13 +81,13 @@ describe('components/widgets/menu/menu_items/submenu_item', () => {
         );
         wrapper.setState({show: true});
         wrapper.find('#Z').at(1).simulate('click');
-        await expect(action1).toHaveBeenCalledTimes(1);
+        await expect(action1).toHaveBeenCalledTimes(0);
```

Comment:
Are you sure this should be changing? It looks like it's clicking on the outer menu item which I'd expect to call `action1` with your changes

Reply:
Sorry for the delay reviewing this. I meant to look more into why this changed, but it dropped off my radar. I'll try to take a look this week